src/tqdm_publisher/_demo/_multiple/_server.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import json
+import threading
+import time
+
+import websockets
+
+import tqdm_publisher
+
+
+async def handler(websocket: websockets.WebSocketServerProtocol) -> None:
+    """Handle messages from the client and manage the client connections."""
+
+    class WebSocketProgressBar(threading.Thread):
+
+        def __init__(self, request_id: str):
+            super().__init__()
+            self.request_id = request_id
+
+        def update(self, format_dict) -> None:
+            """
+            This is the function that will run on every update of the TQDM object.
+
+            It will forward the progress to the client.
+            """
+            asyncio.run(
+                websocket.send(message=json.dumps(obj=dict(request_id=self.request_id, format_dict=format_dict)))
+            )
+
+        def run(self):
+            """
+            Emulate running the specified number of tasks by sleeping the specified amount of time on each iteration.
+
+            Defaults are chosen for a deterministic and regular update period of one second for a total time of one minute.
+            """
+            all_task_durations_in_seconds = [0.1 for _ in range(100)]  # Ten seconds of one hundred tasks
```

Comment:
```suggestion
            all_task_durations_in_seconds = [1.0 for _ in range(10)]  # Ten seconds at one task per second
```

I had to decrease this update rate (unusual to receive updates faster than that) otherwise the webpage would get overloaded and wouldn't update the bar in real time

Reply:
Hmmm alright.